kubernetes/deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: module-14
+spec:
+  selector:
+    matchLabels:
+      app: module-14
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: module-14
+    spec:
+      containers:
+        - name: todo-app
+          image: todo-app:prod
+          imagePullPolicy: Never
+          env:
+            - name: FLASK_APP
+              value: 'todo_app/app'
+            - name: FLASK_ENV
+              value: 'production'
+            - name: SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: secrets
```

Comment:
According to the instructions, all secret values should be part of a single secret "todo-app". This is assigned to "secrets" secret instead